consensus/validation.go

```diff
@@ -760,6 +760,8 @@ func validateV2FileContracts(ms *MidState, txn types.V2Transaction) error {
 			return fmt.Errorf("does not increase revision number (%v -> %v)", cur.RevisionNumber, rev.RevisionNumber)
 		case !revOutputSum.Equals(curOutputSum):
 			return fmt.Errorf("modifies output sum (%d H -> %d H)", curOutputSum, revOutputSum)
+		case rev.MissedHostValue.Cmp(cur.MissedHostValue) > 0:
+			return fmt.Errorf("has missed host value (%d H) exceeding old value (%d H)", rev.MissedHostValue, cur.MissedHostValue)
```

Comment:
good catch. MissedHost should be free to decrease (meaning: the host has put up more collateral), but not increase -- the host shouldn't be able to rescind the collateral they put up.